core/src/main/scala/org/apache/spark/api/r/RBackend.scala

```diff
@@ -29,6 +29,7 @@ import io.netty.channel.socket.nio.NioServerSocketChannel
 import io.netty.handler.codec.LengthFieldBasedFrameDecoder
 import io.netty.handler.codec.bytes.{ByteArrayDecoder, ByteArrayEncoder}
 
+import org.apache.spark.SparkConf
 import org.apache.spark.Logging
```

Comment:
super minor, but these two imports can be merged. I can fix this when I merge this.